scripts/export_tasks.py

```diff
@@ -0,0 +1,40 @@
+import ast
+import os
+import pathlib
+import json
+
+lib_to_task_map = {}
+
+
+def extract_tasks(library_name, variable_name, value):
```

Comment:
Would be great if you could add docstrings in these methods. What is `variable_name` and what is `value`? Would also be good to add some typings